client/containers/Pub/PubDocument/PubBottom/Notes.tsx

```diff
@@ -109,11 +102,11 @@ const Note = (props: NoteProps) => {
 
 const Notes = (props: NotesProps) => {
 	const { notes, ...restProps } = props;
+	const hasNumbering = notes.some(({ number }) => !!number);
 	return (
-		<ul className="notes-component">
+		<ul className={classNames('notes-component', !hasNumbering && 'no-padding')}>
```

Comment:
I'd suggest `has-numbering` instead. Helps when you're looking at the SCSS side of the code and you see:

```
.has-numbering { padding: 0; }
```
instead of 
```
.no-padding { padding: 0; }
```
and you get some context about why this class exists at all, without having to look elsewhere.